code/src/components/WelcomeMessagePhone.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export const WelcomeMessagePhone = ({ phone, setPhone }) => {
+  const handlePhoneChange = (event) => {
+    setPhone(event.target.value);
+    console.log('setPhone', event.target.value)
+  }
+
+  return (
+    <div className="phoneNumber">
+      <p>What is your phone number??</p>
+      <input type="tel" placeholder="Your phone number" value={phone} onChange={handlePhoneChange} />
+    </div>
+  )
+}
```

Comment:
Could this component be combined with the Name component?

Reply:
I think it could be, I had some issues making it and decided to make it separately